conanfile.py

```diff
@@ -35,6 +35,7 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
         self.options["fmt"].header_only = True
+        self.options["boost"].header_only = True
```

Comment:
Is the `boost::header_only` option absolutely necessary? For conan center packages it is required that only absolutely necessary options to other packages are set. 

Reason is: The option is transitively applied and must be explicitly override in consuming conan files. E.g., if some one wants to use `boost::log=True` they must also explicitly set `boost::header_only=False`.

Reply:
Oh, I didn't know that. I'll just revert that then